### ISSUE TITLE
command+f error

### ISSUE THREAD

**Issue by lingyun000** [singerdmx/flutter-quill#1636]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/singerdmx/flutter-quill/issues)

### Flutter Quill version

9.0+

### Steps to reproduce

command+f

### Expected results

Show search box

### Actual results

<img width="2263" alt="Xnip2023-12-21_16-27-17" src="https://github.com/singerdmx/flutter-quill/assets/38176447/759beece-1b64-4cc1-a6a8-37e97d0b7691">

### Code sample

<details><summary>Code sample</summary>

```dart
Restarted application in 654ms.

════════ Exception caught by widgets library ═══════════════════════════════════
The following UnimplementedError was thrown building QuillToolbarSearchDialog(dirty, dependencies: [_LocalizationsScope-[GlobalKey#afe35]], state: QuillToolbarSearchDialogState#2755b):
The instance of FlutterQuillLocalizations.of(context) is null and it's required, please make sure you wrapping the current widget with FlutterQuillLocalizationsWidget or add FlutterQuillLocalizations.delegate to the localizationsDelegates in your App widget, please consider report this in GitHub as a bug

The relevant error-causing widget was:
    MaterialApp MaterialApp:file:///Users/wangji/Downloads/flutter-quill-master/example/lib/main.dart:48:18

When the exception was thrown, this was the stack:
#0      LocalizationsExt.loc (package:flutter_quill/src/l10n/extensions/localizations.dart:13:10)
#1      QuillToolbarSearchDialogState.build (package:flutter_quill/src/widgets/toolbar/buttons/search/search_dialog.dart:124:32)
#2      StatefulElement.build (package:flutter/src/widgets/framework.dart:5583:27)
#3      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5471:15)
#4      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#5      Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#6      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5453:5)
#7      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5625:11)
#8      ComponentElement.mount (package:flutter/src/widgets/framework.dart:5447:5)
...     Normal element mounting (205 frames)
#213    Element.inflateWidget (package:flutter/src/widgets/framework.dart:4326:16)
#214    MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:6871:36)
#215    Element.updateChild (package:flutter/src/widgets/framework.dart:3837:18)
#216    Element.updateChildren (package:flutter/src/widgets/framework.dart:4024:32)
#217    MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6896:17)
#218    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#219    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#220    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#221    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#222    StatefulElement.update (package:flutter/src/widgets/framework.dart:5657:5)
#223    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#224    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#225    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#226    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#227    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#228    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#229    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#230    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#231    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:105:11)
#232    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#233    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#234    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#235    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#236    StatefulElement.update (package:flutter/src/widgets/framework.dart:5657:5)
#237    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#238    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#239    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#240    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#241    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:105:11)
#242    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#243    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#244    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#245    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#246    StatefulElement.update (package:flutter/src/widgets/framework.dart:5657:5)
#247    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#248    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#249    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#250    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#251    StatefulElement.update (package:flutter/src/widgets/framework.dart:5657:5)
#252    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#253    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6743:14)
#254    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#255    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6743:14)
#256    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#257    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#258    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#259    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#260    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#261    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#262    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#263    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#264    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#265    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#266    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#267    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#268    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2895:19)
#269    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:984:21)
#270    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:457:5)
#271    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1325:15)
#272    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1255:9)
#273    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1113:5)
#274    _invoke (dart:ui/hooks.dart:312:13)
#275    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:383:5)
#276    _drawFrame (dart:ui/hooks.dart:283:31)

════════════════════════════════════════════════════════════════════════════════

```

</details>

### Logs

<details><summary>Logs</summary>

```console
Restarted application in 654ms.

════════ Exception caught by widgets library ═══════════════════════════════════
The following UnimplementedError was thrown building QuillToolbarSearchDialog(dirty, dependencies: [_LocalizationsScope-[GlobalKey#afe35]], state: QuillToolbarSearchDialogState#2755b):
The instance of FlutterQuillLocalizations.of(context) is null and it's required, please make sure you wrapping the current widget with FlutterQuillLocalizationsWidget or add FlutterQuillLocalizations.delegate to the localizationsDelegates in your App widget, please consider report this in GitHub as a bug

The relevant error-causing widget was:
    MaterialApp MaterialApp:file:///Users/wangji/Downloads/flutter-quill-master/example/lib/main.dart:48:18

When the exception was thrown, this was the stack:
#0      LocalizationsExt.loc (package:flutter_quill/src/l10n/extensions/localizations.dart:13:10)
#1      QuillToolbarSearchDialogState.build (package:flutter_quill/src/widgets/toolbar/buttons/search/search_dialog.dart:124:32)
#2      StatefulElement.build (package:flutter/src/widgets/framework.dart:5583:27)
#3      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5471:15)
#4      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#5      Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#6      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5453:5)
#7      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5625:11)
#8      ComponentElement.mount (package:flutter/src/widgets/framework.dart:5447:5)
...     Normal element mounting (205 frames)
#213    Element.inflateWidget (package:flutter/src/widgets/framework.dart:4326:16)
#214    MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:6871:36)
#215    Element.updateChild (package:flutter/src/widgets/framework.dart:3837:18)
#216    Element.updateChildren (package:flutter/src/widgets/framework.dart:4024:32)
#217    MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6896:17)
#218    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#219    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#220    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#221    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#222    StatefulElement.update (package:flutter/src/widgets/framework.dart:5657:5)
#223    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#224    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#225    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#226    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#227    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#228    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#229    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#230    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#231    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:105:11)
#232    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#233    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#234    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#235    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#236    StatefulElement.update (package:flutter/src/widgets/framework.dart:5657:5)
#237    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#238    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#239    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#240    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#241    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:105:11)
#242    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#243    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#244    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#245    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#246    StatefulElement.update (package:flutter/src/widgets/framework.dart:5657:5)
#247    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#248    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#249    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#250    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#251    StatefulElement.update (package:flutter/src/widgets/framework.dart:5657:5)
#252    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#253    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6743:14)
#254    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#255    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6743:14)
#256    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#257    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#258    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#259    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#260    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#261    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#262    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#263    ProxyElement.update (package:flutter/src/widgets/framework.dart:5800:5)
#264    Element.updateChild (package:flutter/src/widgets/framework.dart:3815:15)
#265    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5496:16)
#266    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5634:11)
#267    Element.rebuild (package:flutter/src/widgets/framework.dart:5187:7)
#268    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2895:19)
#269    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:984:21)
#270    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:457:5)
#271    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1325:15)
#272    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1255:9)
#273    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1113:5)
#274    _invoke (dart:ui/hooks.dart:312:13)
#275    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:383:5)
#276    _drawFrame (dart:ui/hooks.dart:283:31)

════════════════════════════════════════════════════════════════════════════════

```

</details>

**Comment by singerdmx** [comment]
Hello there, on behalf the Flutter Quill Team I would like to thank you for opening your first issue. Your inputs and insights are valuable in shaping a stable and reliable version for all our users. Thank you for being part of the open-source community!

**Comment by ellet0** [comment]
Fixed, in `9.1.1`

to avoid such issues in production apps in the future add the localizations delegate of flutter quill in your app
